packages/forms-web-app/__tests__/unit/controllers/examination/enter-comment/controller.test.js

```diff
@@ -3,158 +3,126 @@ const {
 	postEnterComment
 } = require('../../../../../src/controllers/examination/enter-comment/controller');
 
-const {
-	getRedirectUrl
-} = require('../../../../../src/controllers/examination/enter-comment/utils/get-redirect-url.js');
-
 let {
-	addKeyValueToActiveSubmissionItem,
-	getActiveSubmissionItem,
-	getSubmissionItemSubmissionType
+	addKeyValueToActiveSubmissionItem
 } = require('../../../../../src/controllers/examination/session/submission-items-session');
+let {
+	getPageData
+} = require('../../../../../src/controllers/examination/enter-comment/utils/get-page-data');
+let {
+	getRedirectRoute
+} = require('../../../../../src/controllers/examination/enter-comment/utils/get-redirect-route');
+let {
+	getSubmissionItemPageUrl
+} = require('../../../../../src/controllers/examination/utils/get-submission-item-page-url');
 
 jest.mock('../../../../../src/controllers/examination/session/submission-items-session', () => ({
-	addKeyValueToActiveSubmissionItem: jest.fn(),
-	getActiveSubmissionItem: jest.fn(),
-	getSubmissionItemSubmissionType: jest.fn()
+	addKeyValueToActiveSubmissionItem: jest.fn()
+}));
+jest.mock('../../../../../src/controllers/examination/enter-comment/utils/get-page-data', () => ({
+	getPageData: jest.fn()
 }));
-
 jest.mock(
-	'../../../../../src/controllers/examination/enter-comment/utils/get-redirect-url.js',
+	'../../../../../src/controllers/examination/enter-comment/utils/get-redirect-route',
 	() => ({
-		getRedirectUrl: jest.fn()
+		getRedirectRoute: jest.fn()
 	})
 );
+jest.mock('../../../../../src/controllers/examination/utils/get-submission-item-page-url', () => ({
+	getSubmissionItemPageUrl: jest.fn()
+}));
 
 describe('controllers/examination/enter-comment/controller', () => {
-	const mockSession = 'mock session';
+	const req = {
+		body: {},
+		session: {},
+		query: {}
+	};
 	const res = {
 		redirect: jest.fn(),
 		render: jest.fn(),
 		status: jest.fn(() => res)
 	};
-	const req = {
-		session: mockSession
-	};
-	const mockComment = 'Comment';
-	const mockSubmissionItem = 'Submission item';
-	const pageData = {
-		activeSubmissionItemTitle: mockSubmissionItem,
-		backLinkUrl: '/examination/select-upload-evidence-or-comment',
-		comment: '',
-		id: 'examination-enter-comment',
-		pageTitle: 'Your comment',
-		title: 'Your comment'
-	};
 	describe('#getEnterComment', () => {
-		describe('When rendering the enter comment page', () => {
+		describe('When making a get request', () => {
```

Comment:
this is render a page not a get request ? 

hard when the post is post. But the post will - render or redirect